### PR TITLE
chore(helm): upgrade microservice-chart to version 5.11.0 and create a Hikari beta version

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### List of Changes

🆙 Bump to Helm Chart `v5.11.0`
🚰 new Github Actions to handle canary v5

#### Motivation and Context

[Jira Issue](https://pagopa.atlassian.net/browse/PAGOPA-2502)

#### How Has This Been Tested?

✋ manually
 [Release And Deploy](https://github.com/pagopa/pagopa-debt-position/actions/workflows/release_deploy.yml)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: helm/values-uat.yaml [hunk 1]
```diff
@@ -138,21 +138,17 @@ microservice-chart:
...
+      type: header
+      headerName: X-Canary
+      headerValue: canary
+      weightPercent: 0
+    image:
+      repository: ghcr.io/pagopa/pagopa-debt-position
+      tag: "0.11.63-3-hikari"
+      pullPolicy: Always
+    envConfig:
+      SPRING_DATASOURCE_URL: "jdbc:postgresql://gpd-db.u.internal.postgresql.pagopa.it:5432/apd?sslmode=require&prepareThreshold=0&tcpKeepAlive=true"
```
Comment (jacopocarlini) [inline helm/values-uat.yaml:152]: @cap-ang il `tcpKeepAlive` è settato per DEV e UAT, ma non in PRODUZIONE. Sono configurazione volute?

Comment (cap-ang) [reply helm/values-uat.yaml:152]: > The keepalive mechanism above is ideal for long-standing idle connections [...] Unless you know you'll have long-lived idle connections, and that your backend (or network equipment) will interfere with these connections, it's better to leave this off.
from https://www.npgsql.org/doc/keepalive.html

Based on this doc, I'd set it to false (default value), agree?

**Review by cap-ang — APPROVED** [review]
LGTM, how it was tested?

**Comment by jacopocarlini** [comment]
> LGTM
> 
> the canary and primary deployment can be deployed individually?

yes

**Pull request merged**